pac4j-core/src/test/java/org/pac4j/core/util/CommonHelperTests.java

```diff
@@ -146,6 +146,11 @@ public void testAreEqualsIgnoreCaseAndTrimFails() {
         assertFalse(CommonHelper.areEqualsIgnoreCaseAndTrim(VALUE, null));
         assertFalse(CommonHelper.areEqualsIgnoreCaseAndTrim(NAME, VALUE));
     }
+    
+    @Test
+    public void testClassPathPrefix() {
```

Comment:
Mainly I thought your other tests were covering most of my code changes already especially since the paths are now converted to resources in code.